src/promql/src/planner.rs

```diff
@@ -139,14 +141,56 @@ impl<S: ContextProvider> PromPlanner<S> {
                 name: "Prom Unary Expr",
             }
             .fail()?,
-            PromExpr::Binary(PromBinaryExpr { lhs, rhs, .. }) => {
-                let _left_input = self.prom_expr_to_plan(*lhs.clone())?;
-                let _right_input = self.prom_expr_to_plan(*rhs.clone())?;
-
-                UnsupportedExprSnafu {
-                    name: "Prom Binary Expr",
+            PromExpr::Binary(PromBinaryExpr { lhs, rhs, op, .. }) => {
+                match (
+                    Self::try_build_literal_expr(lhs),
+                    Self::try_build_literal_expr(rhs),
+                ) {
+                    (Some(_lhs), Some(_rhs)) => {
+                        todo!("is this a legal case?")
```

Comment:
I think it's a legal case; you can execute `1+1` in Prometheus too.

Reply:
https://github.com/GreptimeTeam/greptimedb/pull/889#discussion_r1089860423